docs/wagtail-pages.md

```diff
@@ -0,0 +1,207 @@
+# Wagtail pages
+
+[Wagtail pages](http://docs.wagtail.io/en/v1.13.4/topics/pages.html) are 
+[Django models](https://docs.djangoproject.com/en/1.11/topics/db/models/) 
+that are constructed of 
+[fields](#fields), [StreamFields](#streamfields), and [panels](#panels) 
+that are rendered in a standard way. 
+All CFPB Wagtail pages should inherit from the 
+[`v1.models.base.CFGOVPage` class](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L60).
```

Comment:
I'm not sure of a better way to do this, but the `#L60` is likely to get out of date here. Potentially you could link to a particular commit, but I'm not sure that's better.

Reply:
Yeah, I'm not sure. Maybe if we generated API docs for CFGOVPage it'd be better (ala #4706).

Reply:
Another possible thought would be to clean up [models/base.py](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py) so that the file itself is a satisfactory thing to link to, without the page number. Currently most of that file is CFGOVPage, with a few other odds and ends that could be relocated.

Reply:
For now I'll just link to `models/base.py`... that gets anyone who's curious close enough I think.